style.css

```diff
@@ -0,0 +1,41 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: hsl(41, 70%, 94%);
+  color: hsl(41, 70%, 20%);
+  font-family: "Libre Baskerville", serif;
+}
+
+h1 {
+  background-color: white;
+  color: rgb(31, 70, 122);
```

Comment:
Consider sticking with _either_ rgb or hsl color values -- mixing them up makes it harder to know which colors match. See [CSS best practices](https://mica-web.github.io/learn/css/best-practices#use-color-values-logically-and-consistently).